vocab/security/vocabulary.yml

```diff
@@ -215,51 +252,171 @@ class:
         url: https://w3c-ccg.github.io/ld-cryptosuite-registry/#ed25519
 
 property:
+  - id: verificationMethod
+    label: Verification method
+    range: sec:VerificationMethod
+    comment: A `verificationMethod` property is used to specify a URL that contains information used for proof verification.
+    see_also:
+      - label: Decentralized Identifiers (DIDs) v1.0
+        url: https://www.w3.org/TR/did-core/#verification-methods
+
+  - id: domain
+    label: Domain of a proof
+    domain: sec:Proof
+    range: xsd:string
+    comment: The `domain` property is used to associate a domain with a proof, for use with a `proofPurpose` such as `authentication` and indicating its intended usage.
+
+  - id: challenge
+    label: Challenge with a proof
+    domain: sec:Proof
+    range: xsd:string
+    comment: The challenge property is used to associate a challenge with a proof, for use with a `proofPurpose` such as `authentication`. This string value SHOULD be included in a proof if a `domain` is specified.
+
+  - id: proofPurpose
+    label: Proof purpose
+    domain: sec:Proof
+    range: xsd:string
+    comment: The` proofPurpose` property is used to associate a purpose, such as `assertionMethod` or `authentication` with a proof. The proof purpose acts as a safeguard to prevent the proof from being misused by being applied to a purpose other than the one that was intended.
+
+  - id: proofValue
+    label: Proof value
+    domain: sec:Proof
+    range: xsd:string
+    comment: A string value that contains the data necessary to verify the digital proof using the `verificationMethod` specified
+
+  - id: proof
+    label: Proof sets
+    range: sec:ProofGraph
+    comment: The value of the `proof` property MUST identify <a href=”#ProofGraph”>`ProofGraph` instances</a> (informally, it indirectly identifies <a href=”#Proof”>`Proof` instances</a> each contained in a separate graph). The property is used to associate a proof with a graph of information. The proof property is typically not included in the canonicalized graphs that are then digested, and digitally signed. The order of the proofs is not relevant.
```

Comment:
```suggestion
    comment: The value of the `proof` property MUST identify <a href=”#ProofGraph”>`ProofGraph` instances</a> (informally, it indirectly identifies <a href=”#Proof”>`Proof` instances</a>, each contained in a separate graph). The property is used to associate a proof with a graph of information. The proof property is typically not included in the canonicalized graphs that are then digested and digitally signed. The order of the proofs is not relevant.
```

Reply:
To allow progression, I will merge this (and similar) editorial changes as proposed by @TallTed, so that we could merge this PR in a subsequent step. We can always come back and wordsmith the definitions.